cmd/collection/main.go

```diff
@@ -122,6 +123,8 @@ func main() {
 				"additional fraction of replica timeout that the primary will wait for votes")
 			flags.DurationVar(&blockRateDelay, "block-rate-delay", 250*time.Millisecond,
 				"the delay to broadcast block proposal in order to control block production rate")
+			flags.BoolVar(&rpcMetricsEnabled, "rpc-metrics-enabled", false,
```

Comment:
Any reason we default to false? I'd think we'd want to default to metrics on.

Reply:
mainly because it was mentioned [here](https://github.com/grpc-ecosystem/go-grpc-prometheus/blob/82c243799c991a7d5859215fba44a81834a52a71/server.go#L42) that metrics historgram is expensive